examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>the link to api docs should ideally be:</p><p> </p><pre class="ql-syntax" spellcheck="false">{class}link for the documentation &lt;pymc3.distributions.multivariate.LKJCholeskyCov&gt;
</pre><p> </p><p>by doing this, if we restructure the api page or get multiversion docs both of which will probably change the exact url, we only need to rebuild the documentation for all the links to be fixed instead of having to track down all the urls throughout the notebooks.</p><p> </p><p>The link to the other notebook should probably be a <code>{doc}</code>  or <code>{ref}</code>  object, but I'm not yet sure which approach to recommend.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Sorry, I do not understand how to implement the link with your suggested method. Can you link a notebook with an example of this approach?</p>

Reply:
We recently changed the notebook renderer from nbsphinx to myst-nb which is built on top of the myst parser which is extended markdown to basically support all of rST features. We can therefore use any sphinx directive and role from the notebooks. I added some guidance on sphinx hyperlinks using either markdown or rst at 

As for examples, https://arviz-devs.github.io/arviz/user_guide/pymc3_refitting_xr_lik.html has at least one of these intersphinx links and https://github.com/pymc-devs/pymc3/blob/init_glossary/docs/source/developer_guide_implementing_distribution.md is pure markdown but has multiple examples and there are no rendering differences between the markdown in notebooks and markdown files anymore, everything used there can also be used in a markdown cell of a notebook

Reply:
Sorry, I do not know how to compile these commands to check if they work correctly, referring also your previous comment on  sphinx labels (I never used sphinx). Can you suggest a source for this?

Reply:
<p>Resources to learn sphinx are historically bad and impossible to find. I know there are a lot of people working on that so there should be great tutorials coming up soon, but I think they are still WIP. In my opinion however, it's much more important to learn myst which is what we'll basically use here and/or rST (used in docstrings) than it is to actually learn sphinx itself. The docs here are already set and configure to be built with more or less everything we want so changes to sphinx configuration or it's extensions are quite rare.</p><p> </p><p>I like https://myst-parser.readthedocs.io/en/latest/ but I was already relatively familiar with rST so I might be a bit biased. Let me know what you think about these docs. </p><p> </p><p>Regarding this specific case, which is also quite central and common when writing docs, what I am suggesting is to update the "raw" link that currently looks like:</p><p> </p><pre class="ql-syntax" spellcheck="false">[link for the documentation](https://docs.pymc.io/api/distributions/multivariate.html#pymc3.distributions.multivariate.LKJCholeskyCov)
</pre><p> </p><p>for a cross-reference (very similar to a latex <code>\ref</code>  for example). As this is a cross reference, it won't be rendered by the markdown preview, again, like latex, it will only be rendered after building the whole documentation with sphinx (we have some docs at https://github.com/pymc-devs/pymc3/wiki/Generate-Documentation but hopefully all that will soon be better documented and overall easier). </p><p> </p><p>For now, I would live dangerously and trust my gut that things will eventually render correctly. </p><p> </p><p>In myst, cross-references use the following patterns:</p><p> </p><pre class="ql-syntax" spellcheck="false">{reference_type}<code>reference_target</code>
# or
{reference_type}<code>custom text &lt;reference_target&gt;</code>
</pre><p> </p><p>here I think we should use the 2nd option with the custom text. Our target is the LKJ class and as we are defining the target as the import path we have to set the reference type to class.</p>

Reply:
Thanks, extremely clear explanations! I gave it a go adding reference_types: func, class and doc. 

Remarks:
1. for [this notebook reference](https://docs.pymc.io/pymc-examples/examples/diagnostics_and_criticism/posterior_predictive.html) I tried some nonsense with 
```
{doc}`posterior predictive samples <pymc-examples.examples.diagnostics_and_criticism.posterior_predictive>`
```
2.  I could not guess the reference_target of [this notebook](https://docs.pymc.io/notebooks/LKJ.html).

Reply:
I answered the 1st point below before seeing this.

Regarding the 2nd point, `doc` type references use paths and those can either be relative to the home of the docs or to the current file. I find relative to the home of the docs much better, so it would be `` {doc}`case_studies/LKJ` ``

reference https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#cross-referencing-using-roles

Reply:
Awesome thanks!  Just noting that `{doc}`case_studies/LKJ` did not catch the home of the docs folder, while `{doc}`/case_studies/LKJ` did (also for the other doc reference in the notebook).